domenetjenester/arbeidsforhold/src/main/java/no/nav/foreldrepenger/domene/arbeidInntektsmelding/ArbeidsforholdInntektsmeldingStatus.java

```diff
@@ -0,0 +1,18 @@
+package no.nav.foreldrepenger.domene.arbeidInntektsmelding;
+
+import jakarta.validation.constraints.NotNull;
+import no.nav.foreldrepenger.behandlingslager.virksomhet.Arbeidsgiver;
+import no.nav.foreldrepenger.domene.typer.InternArbeidsforholdRef;
+
+public record ArbeidsforholdInntektsmeldingStatus(@NotNull Arbeidsgiver arbeidsgiver, @NotNull InternArbeidsforholdRef ref, @NotNull InntektsmeldingStatus inntektsmeldingStatus){
+    @Override
+    public String toString() {
+        return "ArbeidsforholdMangel{" + "arbeidsgiver=" + arbeidsgiver + ", ref=" + ref + ", status=" + inntektsmeldingStatus + '}';
+    }
+    public enum InntektsmeldingStatus {
+        MOTTATT,
+        IKKE_MOTTAT,
+        AVKLART_IKKE_PÅKREVD,
```

Comment:
Har vi behov for avklart ikke påkrevd?

Reply:
> Har vi behov for avklart ikke påkrevd?

Jeg ser at du bruker den så glem mitt spm

Reply:
Jeg syns det er greit å ha den med som status i intern logikk, det gjør de lettere å forstå også for andre som senere skal jobbe med.